database/migrations/20220610074044-add-status-columns-to-sites.js

```diff
@@ -0,0 +1,49 @@
+module.exports = {
+  up: async (queryInterface, Sequelize) =>
+    queryInterface.sequelize.transaction((t) =>
+      Promise.all([
+        queryInterface.addColumn(
+          "sites", // name of Source model
+          "siteStatus", // name of column we're adding
+          {
+            type: Sequelize.ENUM,
+            values: ["INIT", "LAUNCH", "LIVE"],
+            allowNull: false,
+            transaction: t,
+          }
+        ),
+        queryInterface.addColumn(
+          "sites", // name of Source model
+          "jobStatus", // name of column we're adding
+          {
+            type: Sequelize.ENUM,
+            values: ["READY", "RUNNING", "FAILED"],
+            allowNull: false,
+            transaction: t,
+          }
+        ),
+      ])
+    ),
```

Comment:
Mismatch in the name of columns in `down` - we should set these to `site_status` and `job_status` to be consistent in using snake_case

Reply:
This fix lead me down another rabbit hole. Fixed in 470692a.

### What led to this initial (faulty) code segment
I had initially set the column name to camelCase (`siteStatus` and `jobStatus`) because I ran into an error when running `npm run test` more than once on the database when the column name was snake_case. The error was:
```
[MigrationError: Migration 20220610074044-add-status-columns-to-sites.js (up) failed: Original error: type "enum_sites_site_status" already exists]
```

Anyway, it turns out that what I did had not addressed the root cause of the problem.

### A problem with the way `sequelize.dropAllTables()` handles enums with multi-part column names

We currently depend on `sequelize.dropAllTables();` during our test database clean up.

However, there appears to be a bug with Sequelize - or at least Sequelize has made some assumptions about the enum names for tables with multi-part column names.

Specifically, when Sequelize attempts to drop the sites table, it assumes that the enum names for the `site_status` and `job_status` columns are `enum_sites_siteStatus` and `enum_sites_jobStatus` respectively.

Unfortunately, we have named our enums `enum_sites_site_status` and `enum_sites_job_status` instead.

Hence, we need to add the following SQL statements to ensure that the abovementioned enums are dropped properly.
```
'DROP TYPE IF EXISTS "public"."enum_sites_site_status" CASCADE;'
'DROP TYPE IF EXISTS "public"."enum_sites_job_status" CASCADE;'
```